src/models/user.ts

```diff
@@ -52,4 +52,15 @@ const Profile = mongoose.model(
     })
 )
 
-export { User, Profile }
+const UserRole = mongoose.model(
+  "UserRole",
+  new Schema({
+    name: {
+      type: String,
+      required: true,
+      unique: true,
+    }
+  })
+);
```

Comment:
Does this represent the type of user (admin, trainee,...)? If so, have you confirmed that there's only one field required to create a new role?